Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/CSGSceneBottomGUI.cs

```diff
@@ -13,670 +13,98 @@ namespace Chisel.Editors
 	// TODO: add tooltips
 	public static class CSGSceneBottomGUI
 	{
-		const float kBottomBarHeight    = 18;
-		const float kButtonSpace        = 6;
-		const float kButtonWidth        = 18;
-		const float kDropdownWidth      = kButtonWidth + 8;
+		const float kBottomBarHeight    = 20;
 		const float kTopBarHeight       = 17;
```

Comment:
top bar height is actually not static anymore. It can grow in size when in prefab mode .. we should check if we are, and use a larger value

Reply:
var currentPrefabStage = UnityEditor.Experimental.SceneManagement.PrefabStageUtility.GetCurrentPrefabStage();
if (currentPrefabStage != null)
{
	var prefabRoot = currentPrefabStage.prefabContentsRoot;
	prefabRootTransform = prefabRoot.transform;
	inPrefabMode = (prefabRoot.scene == currentScene);
}